Package.swift

```diff
@@ -70,7 +75,7 @@ if let env = env, String(cString: env, encoding: .utf8) == "1" {
             name: "SentryObjc",
             dependencies: ["SentrySwift"],
             path: "Sources",
-            exclude: ["Sentry/SentryDsn.m", "Sentry/NSLocale+Sentry.m", "Swift", "SentrySwiftUI", "Resources", "Configuration"],
```

Comment:
`h`: Is it really OK to remove `"Sentry/SentryDsn.m", "Sentry/NSLocale+Sentry.m"`?

Reply:
@noahsmartin can you please review and confirm this change?